### PR TITLE
92 user's NFT collection

### PR DESCRIPTION
**Pull request by steinkohl**
This PR implements the users NFT collection.

login in -> http://localhost:3000/user/myCollection

Some images:

image
image

### CODE REVIEW

File: frontend/src/pages/NFTCollection.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			<HeaderTypography style={{ fontSize: "36px", fontWeight: "bold" }}>Your NFT Collection</HeaderTypography>
+			<Grid
+				container
+				spacing={2}
+				alignItems="center"
+				justifyContent="center"
+				direction="column"
+			>
+				<TableCell>
+					<img src={ ExampleUserPic } alt="User Pic"/>
```
Comment (dominic-heil) [inline frontend/src/pages/NFTCollection.js:59]: If you want to make the image look more like in the mockups, add a wrapper ´div´ arround the image, resize the div with the style attributes, give the div a `backgroundColor` and add a `borderRadius` to the div to make it a circle.

File: frontend/src/pages/NFTCollection.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	{ title: "Nürnberg NFT 038", price: "0.0001", nftPageUrl: "/drop", imgUrl: ExampleNftImg38, buttonText: "Join drop" },
+	{ title: "Nürnberg NFT 039", price: "0.0001", nftPageUrl: "/drop", imgUrl: ExampleNftImg39, buttonText: "Join drop", description: "The previous NFT had no description." },
+	{ title: "Nürnberg NFT 040", price: "0.0001", nftPageUrl: "/drop", imgUrl: ExampleNftImg40, buttonText: "Join drop" }
+];
+
+export default function NftCollection() {
+
+	let topPartOfPage = (
+		<Box>
+			<HeaderTypography style={{ fontSize: "36px", fontWeight: "bold" }}>Your NFT Collection</HeaderTypography>
```
Comment (dominic-heil) [inline frontend/src/pages/NFTCollection.js:50]: In the mockup the font size is 22px.

**Review by dominic-heil — CHANGES_REQUESTED** [review]
Decent start :D I added a few things to take a look at.

**Comment by steinkohl** [comment]
Thanks a lot for your feedback!
I will try to implement the changes until meeting.

File: frontend/src/pages/NFTCollection.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+import React from "react";
+// import appwriteApi from "../api/appwriteApi";
+import Grid from "@mui/material/Grid";
+import HeaderTypography from "../components/HeaderTypography";
+import ParagraphTypography from "../components/ParagraphTypography";
+// import { Box, Button, Table, TableBody, TableCell, TableContainer, TableRow } from "@mui/material";
+import { Box, TableCell } from "@mui/material";
+import { secondaryTextColor } from "../assets/jss/colorPalette";
+// import { activeTextColor, secondaryTextColor, textColor } from "../assets/jss/colorPalette";
```
Comment (tuilagio) [inline frontend/src/pages/NFTCollection.js:12]: remove commented out code when you are done :)

File: frontend/src/pages/NFTCollection.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+				<TableCell>
+					<img src={ ExampleUserPic } alt="User Pic"/>
+				</TableCell>
+				<TableCell>
+					Username
+				</TableCell>
+				<RoundedEdgesButton style={{ backgroundColor: "transparent", fontSize: "12px", border: "1px solid #FFFFFF" }} >
+					Edit Profile
+				</RoundedEdgesButton>
+				<Grid direction="row">
```
[the remark this answers is not in the document]
Comment (tuilagio) [reply frontend/src/pages/NFTCollection.js:67]: not related, but I though nested Grid is possible?
https://mui.com/components/grid/#nested-grid

[the remark this answers is not in the document]
Comment (dominic-heil) [reply frontend/src/pages/NFTCollection.js:67]: Ah, didn't know that. Then probably make them a `<Grid container item direction="row">...`

**Comment by dominic-heil** [comment]
Just updated the design to look more like the mockups. Should be mergable now

**Review by dominic-heil — APPROVED** [review]

**Comment by tuilagio** [comment]
I would say we should merge this quickly and move to other important features. Besides, looks perfect to me!

**Review by tuilagio — APPROVED** [review]

**Pull request merged**